src/logging.cc

```diff
@@ -1868,6 +1846,11 @@ void LogMessage::SendToLog() EXCLUSIVE_LOCKS_REQUIRED(log_mutex) {
     if (write(STDERR_FILENO, message, strlen(message)) < 0) {
       // Ignore errors.
     }
+#if defined(__ANDROID__)
+    __android_log_write(ANDROID_LOG_FATAL,
```

Comment:
It is not clear why messages written to logcat are always of fatal severity.

Reply:
Because here is fatal error and is going to print stack trace.

Reply:
Could please add this as comment above then?